Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContainer.js

```diff
@@ -1,2 +1,17 @@
 import React from 'react';
 import './Header.css';
+import HeaderTitle from './HeaderTitle';
+import HeaderContent from './HeaderContent';
+import ImageThumbnail from './ImageThumbnail';
```

Comment:
Good job on getting your imports working. Maybe add an empty line between the imports and your function for cleaner code.